api/src/main/java/com/messagebird/objects/Request.java

```diff
@@ -28,15 +32,28 @@ public Request(String timestamp, String queryParameters, byte[] data) {
         this.data = data;
     }
 
-    String getTimestamp() {
+    public String getTimestamp() {
         return timestamp;
     }
 
-    String getQueryParameters() {
+    public String getQueryParameters() {
```

Comment:
This method is not used with the new `getSortedQueryParameters()`, right? Perhaps it should be removed until we need it later.